igor-web/src/main/java/com/netflix/spinnaker/igor/scm/ManagedDeliveryScmService.java

```diff
@@ -63,10 +65,15 @@ public ManagedDeliveryScmService(
     this.gitHubMaster = gitHubMaster;
     this.gitLabMaster = gitLabMaster;
     this.bitBucketMaster = bitBucketMaster;
-    this.jsonMapper = jsonMapper;
-    this.yamlMapper =
-        new ObjectMapper(new YAMLFactory())
-            .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+    this.jsonMapper = configureMapper(new ObjectMapper());
+    this.yamlMapper = configureMapper(new ObjectMapper(new YAMLFactory()));
+  }
+
+  private static ObjectMapper configureMapper(ObjectMapper mapper) {
+    return mapper
+        .disable(FAIL_ON_UNKNOWN_PROPERTIES)
+        .enable(ACCEPT_CASE_INSENSITIVE_ENUMS)
+        .setSerializationInclusion(NON_NULL);
```

Comment:
These are probably not needed. I copied them from the keel config, but here we just use deserialization into a `Map`, so they shouldn't matter. I decided to construct my own `ObjectMapper`s in this class because I was afraid configuration elsewhere might inadvertently break the parsing here.